apps/condo/pages/_app.tsx

```diff
@@ -82,11 +85,11 @@ const MenuItems: React.FC = () => {
                 />
                 <MenuItem
                     path={'/ticket'}
-                    icon={ThunderboltFilled}
+                    icon={ThunderboltFilledWithBadge}
                     label={'menu.ControlRoom'}
                     disabled={disabled}
                     isCollapsed={isCollapsed}
-                />
+                >{!isCollapsed && <Badge count={ticketsCounter.numberOfUnreadTickets}/>}</MenuItem>
```

Comment:
This solution does't allows to easily reuse count display feature.
Please, introduce `count` property for `MenuItem`, that incapsulates all this stuff.

Reply:
Not sure we need this since only one component need to have badge with count